### PR TITLE
Improve the git hook helper

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Adjust the install hook helper to properly check if a hook is already "linked" and only run the install if the hook isn't linked.

### CODE REVIEW

File: scripts/utils/git [hunk 1]
```diff
@@ -22,36 +22,42 @@ install_hooks() {
   for hook in $1
   do
     # Check if hook is executable
-    if [ -x scripts/${hook} ]
+    if [! -x scripts/${hook} ]
```
Comment (nLight) [inline scripts/utils/git]: Early returns are good 👍

Comment (nLight) [inline scripts/utils/git]: but then spaces are mandatory around an expr so it should be 
`if [ ! -x scripts/${hook} ]` or it throws `[!: command not found`

Comment (orlandohohmeier) [inline scripts/utils/git]: Oh dang, I must have removed the space during the rebase...

Comment (orlandohohmeier) [inline scripts/utils/git]: ✅  Fixed

**Pull request merged**